core/domain/exp_jobs_one_off.py

```diff
@@ -665,57 +662,6 @@ def reduce(key, values):
         yield (key, list(set().union(*final_values)))
 
 
-class ImageDataMigrationJob(jobs.BaseMapReduceOneOffJobManager):
-    """One-off job for migrating the images in the exploration
-    from the GAE to GCS.
-    """
-
-    @classmethod
-    def entity_classes_to_map_over(cls):
-        return [file_models.FileModel]
-
-    @staticmethod
-    def map(file_model):
-        # This job is allowed to run only in Production environment since it
-        # uses GcsFileSystem which can't be used in Development environment.
-        if feconf.DEV_MODE:
-            return
-
-        instance_id = file_model.id
-        filetype = instance_id[instance_id.rfind('.') + 1:]
-        # To separate the image entries from the audio entries we get from the
-        # FileSnapshotContentModel.
-        if filetype in ALLOWED_IMAGE_EXTENSIONS:
-            catched_groups = FILE_MODEL_ID_REGEX.match(instance_id)
-            if not catched_groups:
-                yield (WRONG_INSTANCE_ID, instance_id)
-            else:
-                filename = catched_groups.group(2)
-                exploration_id = catched_groups.group(1)
-                content = file_model.content
-                fs = fs_domain.AbstractFileSystem(
-                    fs_domain.GcsFileSystem(exploration_id))
-                if fs.isfile('image/%s' % filename):
-                    yield (FILE_ALREADY_EXISTS, file_model.id)
-                else:
-                    fs.commit(
-                        'ADMIN', 'image/%s' % filename,
-                        content, mimetype='image/%s' % filetype)
-                    if not fs.isfile('image/%s' % filename):
-                        yield ('Failed to commit file', instance_id)
-                    else:
-                        yield (FILE_COPIED, 1)
-        else:
-            yield ('Invalid filetype', filetype)
-
-    @staticmethod
-    def reduce(status, values):
-        if status == FILE_COPIED or status == FILE_ALREADY_EXISTS:
-            yield (status, len(values))
-        else:
-            yield (status, '%s' % set(values))
-
-
 class ValidationOfImagesOnGCSJob(jobs.BaseMapReduceOneOffJobManager):
```

Comment:
Actually let's get rid of this job too. I realize it is based on FileModel which no longer exists (and replacing FILE_MODEL_ID_REGEX with GCS_IMAGE_ID_REGEX is thus conceptually incorrect). Sorry for not catching this earlier.